cluster-autoscaler/cloudprovider/clusterapi/clusterapi_machineset.go

```diff
@@ -120,3 +122,31 @@ func newMachineSetScalableResource(controller *machineController, machineSet *Ma
 		minSize:    minSize,
 	}, nil
 }
+
+func (r machineSetScalableResource) Labels() map[string]string {
+	return r.machineSet.Spec.Template.Spec.Labels
+}
+
+func (r machineSetScalableResource) Taints() []apiv1.Taint {
+	return r.machineSet.Spec.Template.Spec.Taints
+}
```

Comment:
Nit: These are the opposite way around from in the machinedeployment, very minor but might make updates easier if they diff nicely 😛